### PR TITLE
[grafana-agent] add Grafana Agent

### PR DESCRIPTION
**Pull request by sbaerlocher**
## Description
Addition of the Grafana Agent: https://grafana.com/docs/agent/latest/

## Motivation and Context
Since the Grafana packages are already maintained by you, I thought that your repository for the agent is the right place.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:

- [ ] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [ ] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [ ] All files are up to date with the latest [Contributing Guidelines](https://github.com/mkevenaar/chocolatey-packages/blob/master/CONTRIBUTING.md)
- [ ] The added/modified package passed install/uninstall in the chocolatey test environment.
- [ ] The changes only affect a single package (not including meta package).

### CODE REVIEW

File: automatic/grafana-agent/grafana-agent.nuspec [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    <owners>Maurice Kevenaar, Simon Baerlocher</owners>
+    <!-- ============================== -->
+
+    <!-- == SOFTWARE SPECIFIC SECTION == -->
+    <!-- This section is about the software itself -->
+    <title>Grafana Agent</title>
+    <authors>Grafana Labs</authors>
+    <!-- projectUrl is required for the community feed -->
+    <projectUrl>https://github.com/grafana/agent</projectUrl>
+    <iconUrl>https://cdn.jsdelivr.net/gh/mkevenaar/chocolatey-packages@0462688e26372e1922bd13142072fac888eb9b3f/icons/grafana.svg</iconUrl>
```
Comment (mkevenaar) [inline automatic/grafana-agent/grafana-agent.nuspec:40]: Each package in this repo needs their own icon. A .portable/install package can share their icon. Please add an icon for the Grafana agent. It seems to be different from the Grafana software

File: automatic/grafana-agent/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# <img src="https://cdn.jsdelivr.net/gh/mkevenaar/chocolatey-packages@0462688e26372e1922bd13142072fac888eb9b3f/icons/grafana.svg" width="48" height="48"/> [grafana-agent](https://community.chocolatey.org/packages/grafana-agent)
```
Comment (mkevenaar) [inline automatic/grafana-agent/README.md:1]: This icon needs to be updated with the package icon

File: automatic/grafana-agent/tools/chocolateyinstall.ps1 [hunk 3]
```diff
@@ -0,0 +1,18 @@
+﻿$ErrorActionPreference = 'Stop';
+$PackageParameters = Get-PackageParameters
+
+$toolsDir     = "$(Split-Path -Parent $MyInvocation.MyCommand.Definition)"
+
+$url          = 'https://github.com/grafana/agent/releases/download/v0.26.1/grafana-agent-installer.exe'
```
Comment (mkevenaar) [inline automatic/grafana-agent/tools/chocolateyinstall.ps1:6]: License allows for redistribution, software is less than 200MB in size and the Grafana package containing the binaries. Please make this package embedded, as the other packages.

**Review by mkevenaar — CHANGES_REQUESTED** [review]
Hi @sbaerlocher,

Thank you for submitting this package, I have some small requests that needs to be changed. Could you update these please?

Thanks

File: automatic/grafana-agent/grafana-agent.nuspec (hunk 1 above)
Comment (sbaerlocher) [reply automatic/grafana-agent/grafana-agent.nuspec]: Icon is customized.

File: automatic/grafana-agent/README.md (hunk 2 above)
Comment (sbaerlocher) [reply automatic/grafana-agent/README.md]: Icon is customized.

File: automatic/grafana-agent/tools/chocolateyinstall.ps1 (hunk 3 above)
Comment (sbaerlocher) [reply automatic/grafana-agent/tools/chocolateyinstall.ps1]: Was adjusted that the exe is loaded into the package.

**Review by mkevenaar — APPROVED** [review]
LGTM!

**Comment by mkevenaar** [comment]
@sbaerlocher your changes have been merged, thank you for your contribution!

**Pull request merged**